trade/src/main/java/bisq/trade/Trade.java

```diff
@@ -147,6 +153,7 @@ protected bisq.trade.protobuf.Trade.Builder getTradeBuilder(boolean serializeFor
         Optional.ofNullable(getErrorStackTrace()).ifPresent(builder::setErrorStackTrace);
         Optional.ofNullable(getPeersErrorMessage()).ifPresent(builder::setPeersErrorMessage);
         Optional.ofNullable(getPeersErrorStackTrace()).ifPresent(builder::setPeersErrorStackTrace);
+        Optional.ofNullable(getLifecycleState()).ifPresent(s -> builder.setLifecycleState(s.toProtoEnum()));
```

Comment:
It cannot be null as we set a default value of its not present in the proto. We use the `Optional.ofNullable` pattern only for optional fields.